k8sfv/scale_test.go

```diff
@@ -65,12 +67,39 @@ var _ = Context("with a k8s clientset", func() {
 
 		It("should not leak memory", func() {
 			addNamespaces(clientset, nsPrefix)
+			occupancyMeasurements := []leastsquares.Point{}
 			for ii := 0; ii < 10; ii++ {
```

Comment:
You should probably discard the first couple of measurements since the first runs have the advantage of running in a clean, unfragmented heap.  Also, we should add some way to trigger a GC, such as a SIGUSR.